config/jobs/kubernetes/sig-release/release-branch-jobs/1.25.yaml

```diff
@@ -1752,6 +1754,7 @@ presubmits:
         - --env=CL2_ENABLE_API_AVAILABILITY_MEASUREMENT=true
         - --env=CL2_API_AVAILABILITY_PERCENTAGE_THRESHOLD=99.5
         - --env=KUBEMARK_APISERVER_TEST_ARGS=--max-requests-inflight=80 --max-mutating-requests-inflight=0
+        - --env=KUBEPROXY_TEST_ARGS=--profiling --metrics-bind-address=0.0.0.0
```

Comment:
Why we add these in the other jobs?

Reply:
sorry, I resolved the conversation https://github.com/kubernetes/test-infra/pull/27927#discussion_r1017971041

Basically, this adds a new flag to the preset environment variables that only is present on 1.26 onwards, we have to override it on the jobs that run previous versions or it will fail to run, since that flag doesn't exist at that point in time

Reply:
doing the opposite is much more effort,  adding the new flag to all the new jobs and not forget to add it in the future

Reply:
thanks for all the clarifications